tests/unit/test_cli.py

```diff
@@ -1,9 +1,13 @@
 # coding: utf-8
 
+import dataclasses
 import os
 import shutil
 import tempfile
+from typing import Tuple
 
+import _pytest.monkeypatch
```

Comment:
@happz why monkeypatch fixture is not enough here?

Reply:
It is not necessary to import this package for `monkeypatch` fixture to work, however, when imported, I can use `_pytest.monkeypatch.MonkeyPatch` class in type annotations. `monkeypatch` argument will be an instance of this class, and that helps mypy & my IDE to provide guidance and linting.